README.md

```diff
@@ -173,39 +176,40 @@ pub async fn main(req: Request, env: Env) -> Result<Response> {
 }
 ```
 
-For more information about how to configure these bindings, see: 
-- https://developers.cloudflare.com/workers/cli-wrangler/configuration#keys
-- https://developers.cloudflare.com/workers/learning/using-durable-objects#configuring-durable-object-bindings
+### Supported bindings
+
+* [Plain text](https://developers.cloudflare.com/workers/cli-wrangler/configuration#vars)
+* [Secret text](https://developers.cloudflare.com/workers/cli-wrangler/commands#secret)
+* [KV namespace](https://developers.cloudflare.com/workers/cli-wrangler/configuration#kv_namespaces)
+* [Durable Object class](https://developers.cloudflare.com/workers/learning/using-durable-objects#configuring-durable-object-bindings)
 
-## Durable Objects
 
-### BETA WARNING
-Durable Objects are still in **BETA**, so the same rules apply to the Durable Object code and APIs here in these crates.
+## Using Durable Objects
 
-### Define a Durable Object in Rust
-To define a Durable Object using the `worker` crate you need to implement the `DurableObject` trait 
-on your own struct. Additionally, the `#[durable_object]` attribute macro must be applied to _both_ 
-your struct definition and the trait `impl` block for it.
+> Durable Objects are still in [**BETA**](https://developers.cloudflare.com/workers/runtime-apis/durable-objects), please read the documentation for more details such as limits.
+
+### Define a Durable Object
+
+To define a Durable Object you need to implement the `DurableObject` trait on your own struct. Additionally, the `#[durable_object]` attribute macro must be applied to _both_ your struct definition and the trait `impl` block for it.
 
 ```rust
 use worker::*;
 
 #[durable_object]
-pub struct Chatroom {
+pub struct ChatRoom {
     users: Vec<User>,
     messages: Vec<Message>
```

Comment:
When copy-pasting this code, it would not compile due to missing comma:

```suggestion
    messages: Vec<Message>,
```

Reply:
Nice find!